single_tech_samples/fabric/fabric_ci_cd/docs/option_2/README.md

```diff
@@ -0,0 +1,133 @@
+# Release Pipeline via DevOps Pipeline + REST API calls
+
+This approach leverages REST API calls to make changes to Dev, Stg and Prod Fabric workspaces.
+
+## Using Fabric item APIs for Git integration
+
+Currently, Microsoft Fabric supports Git integration for Azure DevOps only. This article presents a way to use [Fabric REST APIs](https://learn.microsoft.com/rest/api/fabric/articles/using-fabric-apis) to integrate with other GIT source control mechanisms beyond Azure devOps. A brief summary of the steps involved are:
+
+1. Creating a Fabric workspace (if not existing already)
+2. Creating/updating Fabric items in the workspace to reflect what is on the developer branch.
+3. Working as needed in the Fabric workspace.
+4. Updating changes from workspace to reflect them back in source control.
+
+> Note: This sample follows a strategy where each feature branch is paired with a corresponding Fabric workspace, implementing a one-workspace-per-branch approach.
+
+The below picture illustrates these followed by a description of each of the numbered step:
+
+![Fabric CI/CD Architecture](../../images/Option2.png)
+
+This approach assumes that the developer will operate in the following way:
+
+### Pre-setup: Running the PowerShell Scripts
+
+| Script/File | Description |
+|--------|-------------|
+|[params.psd1](../../src/option_2/params.psd1)|Parameters file - used to set the input arguments to the scripts. Update the values as needed.|
+|[update_from_git_to_ws.ps1](../../src/option_2/update_from_git_to_ws.ps1)|Script to create Fabric workspace and sync assets from source control (local git branch) to the workspace.|
+|[update_from_ws_to_git.ps1](../../src/option_2/update_from_ws_to_git.ps1)|Script to update the local repository from item defintions in the Fabric workspace.|
+
+### Recommended Workflow
+
+**Step 0. Prepare for local development**
+- Create new feature branch from `dev` (or any other development branch) and pull new feature branch locally.
```

Comment:
@ydaponte when you say "the dev branch is an AzDO repo" it is not correct. The scripts can work on other source control tools because everything is based on the local development environment. What is true is that the existing devops pipelines on this sample will only work for AzDO. LMK if you have further suggestions, else close the conversation

Reply:
Yes, @elenaterenzi you are right! I would just leave the NOTE then!

Reply:
done, I have left the note at the end of step 0 and slightly reworded. If you have other suggestions feel free to re-open this conversation